google-cloud-logging/test/google/cloud/logging/entry/to_grpc_test.rb

```diff
@@ -71,14 +71,14 @@
     grpc.log_name.must_equal "projects/test/logs/testlog"
 
     grpc.resource.type.must_equal        "webapp_server"
-    Google::Cloud::Core::GRPCUtils.map_to_hash(grpc.resource.labels).must_equal({ "description" => "The server is running in test",
+    Google::Cloud::Logging::Convert.map_to_hash(grpc.resource.labels).must_equal({ "description" => "The server is running in test",
                                                                      "env"         => "test",
                                                                      "valueType"   => "STRING" })
 
     grpc.severity.must_equal :ERROR
     grpc.timestamp.must_equal Google::Protobuf::Timestamp.new(seconds: Time.parse("2016-01-02T03:04:05Z").to_i)
     grpc.insert_id.must_equal "insert123"
-    Google::Cloud::Core::GRPCUtils.map_to_hash(grpc.labels).must_equal("env" => "test", "fizz" => "buzz")
+    Google::Cloud::Logging::Convert.map_to_hash(grpc.labels).must_equal("env" => "test", "fizz" => "buzz")
     grpc.text_payload.must_equal "payload"
     grpc.json_payload.must_be :nil?
     grpc.proto_payload.must_be :nil?
```

Comment:
Want to copy these `Convert.map_to_hash` tests over to Natural Language? I believe its implementation is lacking tests.